app/src/main/res/layout/activity_main.xml

```diff
@@ -4,39 +4,50 @@
     xmlns:tools="http://schemas.android.com/tools"
     tools:context=".MainActivity">
 
+    <data>
+
+        <variable
+            name="viewModel"
+            type="lab.uro.kitori.samplecoroutine.MainViewModel" />
+
+    </data>
+
     <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent">
 
-        <androidx.appcompat.widget.AppCompatTextView
```

Comment:
xmlではAppCompat指定しない方がいいようです

Reply:
正しくは、AppCompatActivity上なら指定しても適切にAppCompatものとして作られる、ということですね
https://vividcode.hatenablog.com/entry/android-app/library/v7-appcompat-compatible-widget-inflation

一例としてandroidxでのAppCompatButtonを見てみるとその旨記載されていました 🙇 
https://developer.android.com/reference/androidx/appcompat/widget/AppCompatButton.html
>This will automatically be used when you use Button in your layouts and the top-level activity / dialog is provided by appcompat. You should only need to manually use this class when writing custom views.